tests/h/search/index_test.py

```diff
@@ -241,6 +241,23 @@ def test_you_can_filter_annotations_by_thread_ids(self, es_client, factories, in
 
         assert SearchResponseWithIDs([annotation2.id]) == response1
 
+    @pytest.mark.parametrize("quote,query", [
+        ("It is a truth universally acknowledged", "truth"),
+        ("यह एक सत्य सार्वभौमिक रूप से स्वीकार किया जाता है", "सत्य"),
+        ("quick brown fox", "QUICK"),
+    ])
+    def test_match_annotation_by_quote(self, factories, index, search, quote, query):
```

Comment:
Can we rename this to be consistent with the other test names in this class? Something like `"test_you_can_search_within_the_quote()"`

Reply:
Done